modules/cli/cmd/logs.go

```diff
@@ -215,7 +215,12 @@ var logsCmd = &cobra.Command{
 		archList, _ := flags.GetStringSlice("arch")
 		nodeList, _ := flags.GetStringSlice("node")
 
-		withTs, _ := flags.GetBool("with-ts")
+		hideTs, _ := flags.GetBool("hide-ts")
+		withTs := true
```

Comment:
Same comment here regarding `hideTs`. You can modify the code to use `!hideTs` instead of defining a new `withTs`.